low-level/files/opt/seagate/sspl/prerequisites/common_config_SN.ini

```diff
@@ -0,0 +1,34 @@
+# single node common configuration
+
+[system_information]
```

Comment:
In current deployment(LDR_R1), provisioner used to keep it as [system_information] only.

Now, SSPL used to have unique kv in [SYSTEM_INFOMATION], just to avoid duplication with [system_information].

Reply:
I dont understand. Why we are still trying to keep duplicated data, given consul we will stop depending upon & depend on hardcoded value in sspl.conf until ConfStor is available. And avoiding confusion with just difference of small & capital letters ?

Reply:
I dont understand. Why we are still trying to keep duplicated data, given consul we will stop depending upon & depend on hardcoded value in sspl.conf until ConfStor is available. And avoiding confusion with just difference of small & capital letters ?

Reply:
Please ignore this file as I'm going to keep sspl.conf.LDR_R2 file with minimal change without this.